pkgs/applications/science/logic/isabelle/components/isabelle-linter.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "isabelle-linter";
-  version = "1.2.1";
+  version = "Isabelle${isabelle.version}-v1.0.0";
```

This doesn't conform with the contributing guide.

I am no sure what upstream is doing but it is weird.

I would suggest the following:

```suggestion
  version = "2023-1.0.0";
```

---

done